cluster/defaults.go

```diff
@@ -554,6 +571,17 @@ func (c *Cluster) setClusterNetworkDefaults() {
 	if c.Network.WeaveNetworkProvider != nil {
 		networkPluginConfigDefaultsMap[WeavePassword] = c.Network.WeaveNetworkProvider.Password
 	}
+	if c.Network.AciNetworkProvider != nil {
```

Comment:
Why is using `networkPluginConfigDefaultsMap` being skipped here like the other providers?

Reply:
Copying to networkPluginConfigDefaultsMap, then to Network.Options[] and checking whether NetworkProvider Option is set and then setting it to Network.Option[] seemed circuitous instead of setting NetworkProviderOption to the default. Besides, moving it to Network.Option seems like polluting the global namespace with CNI specific options, hence followed this path. Please let me know if this is still what you want.

Reply:
Additionally ACI has more parameters that are not options, no defaults can be given to those. In essence, I am combining all parameters with or without defaults under aciNetworkProvider and not using NetworkOptions.

Reply:
Although I couldn't find a hard technical requirement for the `networkPluginConfigDefaultsMap`, I don't want to divert from the use for one plugin, every other plugin uses it so I would like to use the same mechanism for all plugins.